src/client/eppo-client.ts

```diff
@@ -101,7 +109,8 @@ export default class EppoClient implements IEppoClient {
   public getStringAssignment(
     subjectKey: string,
     flagKey: string,
-    subjectAttributes: Record<string, EppoValue> = {},
```

Comment:
Pretty sure we aren't expecting users to define subject attributes as instances of `EppoValue`

If you following this into `getAssignmentInternal` and `findMatchingRule`, `subjectAttributes` is treated as `Record<string, any>` again

https://github.com/Eppo-exp/js-client-sdk-common/blob/fbc31706a156be137f6185947e40f5d164e36b5a/src/rule_evaluator.ts#L16-L17